tests/scripts/task_distributed_test.sh

```diff
@@ -30,7 +30,7 @@ export CUDA_VISIBLE_DEVICES=-1
 
 conda activate ${DGLBACKEND}-ci
 
-python3 -m pip install pytest psutil pyyaml pydantic pandas rdflib ogb filelock || fail "pip install"
+python3 -m pip install pytest psutil pyyaml pydantic pandas rdflib ogb filelock pyarrow || fail "pip install"
```

Comment:
L41 invokes pytest on `tests/tools/*.py`, which may cause issue if the script contains functions starting with `test_`. Can we put all the test files into `tests` folder?